jobs/async-upload/tests/integration/test_integration_async_upload.py

```
@@ -0,0 +1,365 @@
+"""Integration tests for async-upload job functionality."""
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)